htlcswitch/switch.go

```diff
@@ -889,64 +903,80 @@ func (s *Switch) handleLocalResponse(pkt *htlcPacket) {
 	// has been restarted since sending the payment.
 	payment := s.findPayment(pkt.incomingHTLCID)
```

Comment:
If `payment` is nil, we can return already here.

Reply:
Parse failed payment will actually log. Also this will change in upcoming PR.